index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+  <div class="portfolio">
+    <header>
+      <nav class="navbar">
+        <div class="navbottom">
+          <div class="logo">
+            <h2>Carlyle Codes</h2>
+          </div>
+          <div class="menu">
+            <i class="fas fa-bars"></i>
+          </div>
```

Comment:
- [ ] Since this element will be containing your dynamic menu with all of the different sections, I suggest wrapping only this element inside the `<nav>` tag, not the whole top section.